CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [13.0.0](https://github.com/paritytech/substrate-api-sidecar/compare/v12.4.3..v13.0.0) (2022-08-31)
+
+### ⚠ BREAKING CHANGES ⚠
+
+- fix!: integrate @substrate/calc@0.3.0 for partial fees ([#1017](https://github.com/paritytech/substrate-api-sidecar/pull/1017)) ([92e3e1d](https://github.com/paritytech/substrate-api-sidecar/commit/92e3e1daba43bbefb1159fa84ceee0362380ac49))
+    - Note: This removes the query param `feeByEvent` for all `/blocks/*` endpoints. Fee's are correctly calcluated by default and dont require a secondary way of retrieving the `partialFee`. For historical blocks an error might be logged and emmitted for chains that dont have `TransactionPaymentApi::query_feeDetails` in their runtime api. The error is emmitted by polkadot-js regardless of catching it, but it does not affect fee calculation as also logged by a follow up warning. Please refer to the PR for more information.
```

Comment:
```suggestion
    - Note: This removes the query param `feeByEvent` for all `/blocks/*` endpoints. Fee's are correctly calculated by default and don't require a secondary way of retrieving the `partialFee`. For historical blocks, an error might be logged and emitted for chains that don't have `TransactionPaymentApi::query_feeDetails` in their runtime API. The error is emitted by polkadot-js regardless of catching it, but it does not affect fee calculation as also logged by a follow up warning. Please refer to the PR for more information.
```

Reply:
Fixed. I wasn't able to commit this as there was one similar change with James commit above, so I added them manually.